pyccel/codegen/printing/ccode.py

```diff
@@ -1239,6 +1239,8 @@ def _print_Nil(self, expr):
         return 'NULL'
 
     def _print_PyccelAdd(self, expr):
+        if isinstance(expr.args[0].dtype, NativeInteger):
+            self._additional_imports.add('stdint')
```

Comment:
Why do you need `stdint` to do addition? I would have expected you to need it for casting

Reply:
yes, i mentioned it in a previous meeting.